### DISCUSSION TITLE
v2.16.42

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#4882] [General]
## What's Changed
* Enhancement: default to action model in HasActions trait by @awcodes in https://github.com/filamentphp/filament/pull/4844
* Support Column text alignment of the text-start, text-end and text-justify by @AAbosham in https://github.com/filamentphp/filament/pull/4809
* Add missing Arabic translations by @AAbosham in https://github.com/filamentphp/filament/pull/4847
* Rich Editor support for trix-active by @Zagna in https://github.com/filamentphp/filament/pull/4849
* Fix alignment to support RTL and LTR by @AAbosham in https://github.com/filamentphp/filament/pull/4853
* Update ms translations. by @sumardi in https://github.com/filamentphp/filament/pull/4848
* Fix checkbox list without bulk toggleable by @zepfietje in https://github.com/filamentphp/filament/pull/4859
* added missing german translations by @dabenzel in https://github.com/filamentphp/filament/pull/4863
* Update id translations by @margarizaldi in https://github.com/filamentphp/filament/pull/4866
* Fix date time picker duplicate space by @zepfietje in https://github.com/filamentphp/filament/pull/4875
* Fix: undefined $refs.textarea when component tries to render by @awcodes in https://github.com/filamentphp/filament/pull/4881
* Fix: modal issues by @awcodes in https://github.com/filamentphp/filament/pull/4880
* Expose Laravel Loop Object To Column by @Nuhel in https://github.com/filamentphp/filament/pull/4857

## New Contributors
* @dabenzel made their first contribution in https://github.com/filamentphp/filament/pull/4863

**Full Changelog**: https://github.com/filamentphp/filament/compare/v2.16.41...v2.16.42

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v2.16.42'>v2.16.42</a>.</em>

**Comment by intrepidws** [comment]
Since this release, all instances of `TextColumn` in a Relation Manager are appearing with their content centered. If I downgrade back to 2.16.41 - the problem goes away.

**Reply by danharrin** [reply]
Make sure you recompile your theme, clear your view cache, delete overridden views etc. That sounds like a major issue and noone else has reported it in 2 months, so it's probably on your end.

**Reply by intrepidws** [reply]
I had tried all of that (recompile the theme, clear view cache, etc) prior to posting and it wasn't working. After spending far too long trying other solutions, I deleted the `node_modules` directory and that seems to have forced the update for some reason. Anyway, thanks for your help.
